src/Components/missions/Missions.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { toggleJoining } from '../../redux/Missions/missionsReducer';
+import style from './missions.module.css';
+
+function Missions() {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+  return (
+    <>
+      <table className={style.table} data-testid="list-of-missions">
+        <thead className={style.thead}>
+          <tr className={style.header}>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th> </th>
+          </tr>
+        </thead>
+        {missions.map((el) => (
+          <tbody key={el.id} className={style.item}>
+            <td className={style.row}>{el.mission_name}</td>
+            <p className={style.description}>{el.description}</p>
+            <td>
+              <button type="submit" className={el.joined ? style.inactiveMember : style.activeMember}>
+                {
+                el.joined ? 'A MEMBER' : 'Active Member'
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure the texts are correct. The user becomes an "Active Member" after joining the mission. And "Not a member" when leaving the mission, the stats are reversed `A MEMBER` should be `Active Member` and `Active Member` should be `Not a member`

|Your app|Provided design|
|-|-|
|![Screenshot_20221119_084354](https://user-images.githubusercontent.com/25227022/202841110-d4d8b383-c279-4b10-b946-6ea95d1ab097.png)|![Screenshot_20221119_084457](https://user-images.githubusercontent.com/25227022/202841116-96e07db7-67d9-4203-b250-06cf0e5726f2.png)|